pylint/checkers/base/comparison_checker.py

```diff
@@ -81,6 +82,12 @@ class ComparisonChecker(_BasicChecker):
         ),
     }
 
+    @staticmethod
+    def is_singleton_const(node: nodes.NodeNG) -> bool:
+        return isinstance(node, nodes.Const) and any(
```

Comment:
`node.value is in SINGLETON_VALUES`, right?

Reply:
I switched to:
`node.value in SINGLETON_VALUES`